test/api-test.js

```diff
@@ -562,6 +562,12 @@ describe('RippleAPI', function() {
       secret: 'shzjfakiK79YQdMjy4h8cGGfQSV6u'
     };
     assert.doesNotThrow(_.partial(validate.addressAndSecret, goodWallet));
+    assert.throws(_.partial(validate.secret, ''),
+      this.api.errors.ValidationError);
+    //  'passphrase' that starts from 's', like secret
+    assert.doesNotThrow(_.partial(validate.secret, 's!!!'));
```

Comment:
I mean adding `assert.throws(_.partial(validate.secret, 'passphrase'), this.api.errors.ValidationError);`

Code is better than english huh :smiley: 

Also, separate issue is that I think we want:

 `assert.throws(_.partial(validate.secret, 's!!!'), this.api.errors.ValidationError);`